package.json

```diff
@@ -32,13 +32,13 @@
   "size-limit": [
     {
       "path": "packages/components/lib/**/*([a-zA-Z-_]).js",
-      "limit": "70KB",
+      "limit": "140KB",
       "name": "components",
       "webpack": false
     },
     {
       "path": "packages/components/lib/**/*([a-zA-Z-_]).css",
-      "limit": "70KB",
+      "limit": "140KB",
```

Comment:
I'm not sure this config is right.

It seems to indicate that JS and CSS should each be able to be 70kb in size. But the CI failures (for example, see #816) seem like it's the JS + CSS that's putting us over 70kb.